test/transform/CompatibilityTransform.test.js

```diff
@@ -0,0 +1,13 @@
+import assert from 'assert'
+import domino from 'domino'
+import pagelib from '../../build/wikimedia-page-library-transform'
+
+describe('CompatibilityTransform', () => {
+  it('.enableSupport()', () => {
+    const document = domino.createDocument()
+    pagelib.CompatibilityTransform.enableSupport(document)
+    for (const clazz of Object.values(pagelib.CompatibilityTransform.COMPATIBILITY)) {
```

Comment:
Did you mention these sorts of deliberate misspellings (`clazz`) are a pattern in JS?

They make my cheek twitch every time I see them.